cybox/utils/__init__.py

```diff
@@ -73,6 +74,17 @@ def _import_submodules(pkg):
         importlib.import_module(mod_name)
 
 
+def is_sequence(item):
```

Comment:
I understand what you're trying to do, it just feels like an artificial distinction (you can pass in any iterable except a dictionary or string). Since we are converting it to a list anyway, I would be fine forcing people to wrap any generators in a `list()` call. On the other hand, Python's default behavior when iterating over dictionaries is to give you the keys; if someone passes in a dictionary expecting that behavior, I don't know if we should restrict it. IMO, the only thing we really want to check is that it's not a string; if we iterate over that to get characters, we'll have a problem.

That said, I'm fine leaving this; I just saw it and thought "this shouldn't be necessary", but  a quick search didn't find anything better.